app/main/views/questionnaire.py

```diff
@@ -79,10 +80,15 @@ def get_thank_you(eq_id, form_type, period_id, collection_id):
         return redirect("/information/multiple-surveys")
 
     page = render_page('thank-you', questionnaire_manager)
-    questionnaire_manager.delete_user_data()
+    delete_user_data()
     return page
 
 
+def delete_user_data():
+    get_questionnaire_store(current_user.user_id, current_user.user_ik).delete()
+    session_manager.clear()
```

Comment:
Think you've missed a line here- need's to call delete on the questionnaire store once it's retrieved it.